modules/microshift-install-system-requirements.adoc

```diff
@@ -8,15 +8,21 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system-base} or {op-system-ostree}.
+* A compatible version of {op-system-base-full}.For more information, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_microshift/{ocp-version}/html/getting_ready_to_install_microshift/microshift-install-get-ready#get-ready-install-rhde-compatibility-table_microshift-install-get-ready[Compatibility table].
 * AArch64 or x86_64 system architecture.
 * 2 CPU cores.
-* 2 GB RAM for {microshift-short} or 3 GB RAM, required by {op-system-base} for networked-based HTTPs or FTP installations.
+* 2 GB RAM. Installing from the network (UEFI HTTPs or PXE boot) requires 3 GB RAM for {op-system-base}.
 * 10 GB of storage.
 * You have an active {microshift-short} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
-* You have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.
+* When your workload requires Persistent Volumes (PVs), you have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient free capacity for the workloads.
+[NOTE]
```

Comment:
This note is not rendering correctly in the preview, I think it's either because you need a blank line above this one or a +. I'd suggest trying each out and seeing what works for you in a local preview:
```suggestion

[NOTE]
```

Reply:
done